tests/testthat/test-prophet.R

```diff
@@ -0,0 +1,663 @@
+context("Prophet")
+
+# Make sure to run the test for plots with date axes on a system with English language 
+# because months and weekdays will be displayed depending on that
+
+dateTimeVarNames <- c("dateYear", "dateWeek", "dateDay", "timeHour", "timeMinute", "timeSecond")
+dateTimeUnits <- c("years", "weeks", "days", "hours", "mins", "secs")
+
+test_that("Posterior Summary Table results match (linear)", {
+  options <- analysisOptions("Prophet")
+  options$dependent <- "contNormal"
+  options$time <- "dateYear"
+  options$mcmcSamples <- 10
+  options$predictionSavePath <- ""
+  
+  for (i in 1:6) {
+    options$time <- dateTimeVarNames[i]
+    set.seed(1)
+    results <- runAnalysis(name = "Prophet", dataset = "prophetTest.csv", options = options)
```

Comment:
```suggestion
    results <- jaspTools::runAnalysis(name = "Prophet", dataset = "prophetTest.csv", options = options)
```

and the same elsewhere